pubtools/_ami/tasks/push.py

```diff
@@ -448,6 +451,14 @@ def add_args(self):
             default=4,
         )
 
+        group.add_argument(
+            "--snapshot_accounts",
+            help="List of account ids to give snapshot creation permissions to if a new snapshot "
+                 "is created as part of the image push.",
+            type=list,
```

Comment:
How do you expect the 'list' argument to work here?  I'm not sure that it works sensibly.

My understanding of the argparse behavior is that it will more or less do this:

```
parser = arg_definition.type  # in this case, list
parsed_value = parser(raw_string_value)
```

This does not achieve parsing a string value into a list in a sane way.  It will simply produce a list holding one element per character in the string.  I pulled down your change and tried it locally and confirmed the behavior seems to be bogus:

```
$ pubtools-ami-push --snapshot_accounts 679593333241 file:/dev/null
> /home/rmcgover/src/pubtools-ami/pubtools/_ami/tasks/push.py(466)run()
-> if not self.items_in_metadata_service():
(Pdb) pp self.args.snapshot_accounts
['6', '7', '9', '5', '9', '3', '3', '3', '3', '2', '4', '1']
```

I don't think that's the behavior you wanted...

In this family of projects we have a convention for handling list args, https://release-engineering.github.io/pubtools/devguide.html#argument-conventions , so please make it consistent with that.

Also, can you please add at least one test which passes a value for this argument?